serverjs/cubefn.js

```diff
@@ -517,6 +590,16 @@ const methods = {
       pack: draft.initial_state[0][0],
     };
   },
+  newCardAnalytics,
+  getEloAdjustment: (winner, loser) => {
+    const diff = loser - winner;
+    // Expected performance for pick.
+    const expectedA = 1 / (1 + 10 ** (diff / 400));
+    const expectedB = 1 - expectedA;
+    const adjustmentA = 2 * (1 - expectedA);
```

Comment:
This scaling factor needs to be larger for cubes. If card a starts equal elo to card b it takes roughly(approximated with a continuous estimate) 965/scaling comparisons between the two before a's elo has grown by 100. Obviously our implementation would likely do this faster since it compares against multiple cards at once but it still would probably take at least 32/scaling first picks to have it grow by 100. In general to go up x points it takes an exponential number of picks and is inversely related to the scaling factor. It would be a good idea to differentiate cards much quicker since cubes are large enough you won't be able to impact card ratings that much otherwise.